src/components/AddressVerificationContainer/AddressVerificationContainer.module.css

```diff
@@ -1,4 +1,4 @@
-.multiple_address_list {
+.multipleAddressList {
```

Comment:
Isn't the more common convention to use hyphens in CSS class names (i.e. neither the old nor the new version of this)?